src/applications/disability-benefits/all-claims/validations.js

```diff
@@ -480,9 +474,7 @@ export const validateDisabilityName = (
     item => item === itemLowerCased || sippableId(item) === itemSippableId,
   );
   if (itemCount.length > 1) {
-    const errorMessage = showRevisedNewDisabilitiesPage()
-      ? 'You’ve already added this condition to your claim'
-      : 'Please enter a unique condition name';
+    const errorMessage = 'You’ve already added this condition to your claim';
     err.addError(errorMessage);
```

Comment:
not a big deal, but to make this large file 2 lines shorter we can ditch the `errorMessage` vars and just pass into the function